app/src/main/java/github/paroj/dsub2000/util/FileUtil.java

```diff
@@ -69,7 +70,7 @@ public class FileUtil {
     private static final List<String> MUSIC_FILE_EXTENSIONS = Arrays.asList("mp3", "ogg", "aac", "flac", "m4a", "wav", "wma", "opus", "oga");
 	private static final List<String> VIDEO_FILE_EXTENSIONS = Arrays.asList("flv", "mp4", "m4v", "wmv", "avi", "mov", "mpg", "mkv", "3gp", "webm");
 	private static final List<String> PLAYLIST_FILE_EXTENSIONS = Arrays.asList("m3u");
-	private static final int MAX_FILENAME_LENGTH = 254 - ".complete.mp3".length();
+	private static final int MAX_FILENAME_LENGTH = 254 - "00-000-.complete.ext".length();
```

Comment:
we could end up using "flac" as ext, no?

Reply:
You're absolutely right - good point!
I also realized that there was a possibility that existing cached files couldn't be found because they were truncated differently. I've just provided an update to address that.

Additionally, I did some more testing: when going offline, playing a few songs, and then going back online, they were scrobbled correctly.